### PR TITLE
Initial PR

### PR DESCRIPTION
**Pull request by ultiferrago**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,43 @@
-# trustymail
-Scan domains and return data based on trustworthy email best practices
+## Trustworthy Mail  :lock:
+`TrustyMail` is a tool that evaluates SPF/DMARC DNS record configurations to recommend best practices for the records. It saves its results to a CSV.
+
+#### Usage and examples
+
+```bash
+python cli [options] INPUT
```
Comment (h-m-f-t) [inline README.md]: Our instructions should feature running trustymail post-installation (as a module), à la invoking `trustymail` directly. We can also mention running directly, like we have with pshtt. https://github.com/dhs-ncats/pshtt#getting-started

File: trustymail/Domain.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+    def add_mx_record(self, record):
+        self.mx_records.append(record)
+        # Record in format "pref mail_server." Grab only address and remove trailing period.
+        self.mail_servers.append(record.split(" ")[1][:-1])
+
+    def generate_results(self):
+        results = {
+                        "Domain": self.domain_name,
+
```
Comment (h-m-f-t) [inline trustymail/Domain.py]: We need a field for a domain's base domain, similar to https://github.com/dhs-ncats/pshtt#domain-and-redirect-info. Any success with https://publicsuffix.org?

File: trustymail/Domain.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+                        "Sends Mail": self.has_mail(),
+                        "SPF Record": self.has_spf(),
+                        "DMARC Record": self.has_dmarc(),
+
+                        "SPF Results": self.format_list(self.spf),
+                        "DMARC Results": self.format_list(self.dmarc),
+                        "Mail Servers": self.format_list(self.mail_servers),
```
Comment (h-m-f-t) [inline trustymail/Domain.py]: It may make sense for the "Mail Servers" column to be after the "Sends Mail" column.

File: cli.py [hunk 4]
```diff
@@ -0,0 +1,78 @@
+"""TrustyMail A tool for scanning DNS mail records for evaluating security.
+Usage:
+  trustymail (INPUT ...) [options]
+  trustymail (INPUT ...) [--output=OUTFILE] [--timeout=TIMEOUT] [--mx] [--spf] [--dmarc] [--debug]
+  trustymail (-h | --help)
+Options:
```
Comment (h-m-f-t) [inline cli.py]: Can you cover installation of dependancies (via a requirements.txt)? 

We'll post to PyPI once we have a 0.1 and can then update this.

File: trustymail/Domain.py (hunk 2 above)
Comment (ultiferrago) [inline trustymail/Domain.py]: Yes, I need to push those commits to pshtt, but I will add this in to trustymail

File: README.md (hunk 1 above)
Comment (ultiferrago) [inline README.md]: Will do. I used the python run examples as I had not set up the module pip stuff yet. I will do that so we can have it installed as a module.

**Comment by ultiferrago** [comment]
I believe I made all the requested fixes. I also fixed a few other things I happened to see that slipped through.

**Comment by h-m-f-t** [comment]
Live and be free!

**Pull request merged**